pyproject.toml

```diff
@@ -20,6 +20,7 @@ include = [
 [tool.poetry.dependencies]
 python = "^3.9"
 linkml-runtime = "^1.6.2"
+schemasheets = "^0.3.1"
```

Comment:
Why is this being added back in here?

Reply:
schemasheets really is required in this branch, but I didn't mean to add it twice

Reply:
Right, I would have expected not changes to `pyproject.toml` or `poetry.lock` at all in this branch.